ditail.py

```diff
@@ -129,6 +145,8 @@ def tail_files_in(directory):
 def create_argparser():
     parser = argparse.ArgumentParser(description="Prints new lines added to all files in a directory and its subdirectories. Any files added after startup will also be monitored.")
     parser.add_argument("-d", "--debug", action="store_true", help="Enable logging (at DEBUG level)")
+    parser.add_argument("-f", "--filter", action="store", required=False, help="Filter filename (regexp)")
```

Comment:
`required=False` isn't needed. Using option syntax (`-b` `--blah`) implies that the argument is optional, i.e. not required.